btree/btree.c

```diff
@@ -2387,7 +2325,7 @@ static struct ff_head *ff_data(const struct nd *node)
 
 static void *ff_key(const struct nd *node, int idx)
 {
-	struct ff_head *h    = ff_data(node);
+	struct ff_head *h  = ff_data(node);
```

Comment:
Fix the '=' alignment here and also in the below functions.

Reply:
Done.